gtfs2ntfs/src/main.rs

```diff
@@ -58,6 +58,19 @@ struct Opt {
     #[structopt(long = "odt-comment")]
     odt_comment: Option<String>,
 
+    /// If we don’t have the flag activated,
```

Comment:
honestly this description is great, but here is an attempt at another way to describe this (more to be sure to understand correctly, feel free to keep it that way if you prefer)

```
If true, each GTFS `Route` will generate a different `Line`.
Else we group the routes by `agency_id` and `route_short_name` (or `route_long_name` if the short name is empty) and create a `Line` for each group.
```